pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -114,6 +114,15 @@ type GameServerSelector struct {
 	// Players provides a filter on minimum and maximum values for player capacity when retrieving a GameServer
 	// through Allocation. Defaults to no limits.
 	Players *PlayerSelector `json:"players,omitempty"`
+	// (Alpha, CountsAndLists feature flag) Counters provides filters on minimum and maximum values
```

Comment:
Non blocking, but it might be nice to make the format match the ones above - but I don't actually mind 🤷🏻‍♂️ it was a format I 100% made up.

Reply:
I copied the format from pkg/apis/agones/v1/gameserver.go. I can't remember off the top of my head, but I believe @zmerlynn switched to the new comment format for some reason.

Reply:
No biggie. Keep it as is 👍🏻